Assets/HoloToolkit/Utilities/Tests/TextToSpeechManager.unity

```diff
@@ -388,54 +388,6 @@ Light:
   m_BounceIntensity: 1
   m_ShadowRadius: 0
   m_ShadowAngle: 0
---- !u!1 &478672352
```

Comment:
Just ensure you updated your forked branch from Master.
Update the code review if you see any pending merges there.